### PR TITLE
BXMSPROD-1265: master to main leftovers

### PR DESCRIPTION
**Pull request by mbiarnes**
https://github.com/kiegroup/optaplanner/pull/1466
https://github.com/kiegroup/optaweb-employee-rostering/pull/663
https://github.com/kiegroup/optaweb-vehicle-routing/pull/536

<!--
Thank you for submitting this pull request.

### Checklist
- [ ] Documentation updated if applicable.
- [ ] Upgrade recipe provided if applicable.

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* for a <b>pull request build</b> please add comment: <b>Jenkins retest this</b>
* for a <b>specific pull request build</b> please add comment: <b>Jenkins (re)run [optaplanner|apps|examples] tests</b>
* for a <b>full downstream build</b> please add comment: <b>Jenkins run fdb</b>
* for a <b>compile downstream build</b> please add comment: <b>Jenkins run cdb</b>
* for a <b>full production downstream build</b> please add comment: <b>Jenkins execute product fdb</b>
* for an <b>upstream build</b> please add comment: <b>Jenkins run upstream</b>
* for a <b>Quarkus LTS check</b> please add comment: <b>Jenkins run LTS</b>
* for a <b>specific Quarkus LTS check</b> please add comment: <b>Jenkins (re)run [optaplanner|apps|examples] LTS</b>
* for a <b>Native check</b> please add comment: <b>Jenkins run native</b>
* for a <b>specific Native LTS check</b> please add comment: <b>Jenkins (re)run [optaplanner|apps|examples] native</b>
</details>

### CODE REVIEW

**Review by Ginxo — APPROVED** [review]
LGTM

File: optaplanner-docs/src/main/asciidoc/ShadowVariable/ShadowVariable-chapter.adoc [hunk 1]
```diff
@@ -44,7 +44,7 @@ So if A references B, then B references A.
 image::ShadowVariable/bidirectionalVariable.png[align="center"]
 
 For a non-chained planning variable, the bi-directional relationship must be a many to one relationship.
-To map a bi-directional relationship between two planning variables, annotate the master side (which is the genuine side) as a normal planning variable:
+To map a bi-directional relationship between two planning variables, annotate the main side (which is the genuine side) as a normal planning variable:
```
Comment (rsynek) [inline optaplanner-docs/src/main/asciidoc/ShadowVariable/ShadowVariable-chapter.adoc:47]: We should keep the naming consistent with the content in the [`main` branch](https://github.com/kiegroup/optaplanner/blob/main/optaplanner-docs/src/main/asciidoc/ShadowVariable/ShadowVariable-chapter.adoc):

```suggestion
To map a bi-directional relationship between two planning variables, annotate the source side (which is the genuine side) as a normal planning variable:
```

File: optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java [hunk 2]
```diff
@@ -51,7 +51,7 @@
     }
 
     /**
-     * The source variables (masters) that trigger a change to this shadow variable (slave).
+     * The source variables (main) that trigger a change to this shadow variable (slave).
```
Comment (rsynek) [inline optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java:54]: Consistency with the [`main` branch](https://github.com/kiegroup/optaplanner/blob/main/optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java):

```suggestion
     * The source variables (leaders) that trigger a change to this shadow variable (follower).
```

File: optaplanner-benchmark/src/main/resources/org/optaplanner/benchmark/impl/report/twitterbootstrap/js/jquery.js [hunk 3]
```diff
@@ -1261,7 +1261,7 @@ jQuery.extend({
 			// the count of uncompleted subordinates
 			remaining = length !== 1 || ( subordinate && jQuery.isFunction( subordinate.promise ) ) ? length : 0,
 
-			// the master Deferred. If resolveValues consist of only a single Deferred, just use that.
+			// the main Deferred. If resolveValues consist of only a single Deferred, just use that.
```
Comment (triceo) [inline optaplanner-benchmark/src/main/resources/org/optaplanner/benchmark/impl/report/twitterbootstrap/js/jquery.js:1264]: I do not think this is a correct replacement.

File: optaplanner-benchmark/src/main/resources/org/optaplanner/benchmark/impl/report/twitterbootstrap/js/jquery.js [hunk 4]
```diff
@@ -1296,7 +1296,7 @@ jQuery.extend({
 			}
 		}
 
-		// if we're not waiting on anything, resolve the master
+		// if we're not waiting on anything, resolve the main
```
Comment (triceo) [inline optaplanner-benchmark/src/main/resources/org/optaplanner/benchmark/impl/report/twitterbootstrap/js/jquery.js:1299]: I do not think that this is a correct replacement.

File: optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java (hunk 2 above)
Comment (triceo) [inline optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java:54]: I do not think that this is a correct replacement.

File: optaplanner-docs/src/main/asciidoc/ShadowVariable/ShadowVariable-chapter.adoc (hunk 1 above)
Comment (triceo) [inline optaplanner-docs/src/main/asciidoc/ShadowVariable/ShadowVariable-chapter.adoc:47]: I do not think that this is a correct replacement.

**Review by triceo — CHANGES_REQUESTED** [review]
This PR replaces `master` even in places where it does not refer to the Git branch.

**Review by rsynek — CHANGES_REQUESTED** [review]
Thanks @mbiarnes! We should keep the naming consistent with what we have on the `main` branch; otherwise looks good.

File: optaplanner-benchmark/src/main/resources/org/optaplanner/benchmark/impl/report/twitterbootstrap/js/jquery.js (hunk 3 above)
Comment (rsynek) [reply optaplanner-benchmark/src/main/resources/org/optaplanner/benchmark/impl/report/twitterbootstrap/js/jquery.js:1264]: The question is whether we have to change also such an included 3rd party sources.

Comment (triceo) [reply optaplanner-benchmark/src/main/resources/org/optaplanner/benchmark/impl/report/twitterbootstrap/js/jquery.js:1264]: We are replacing the `master` branch.
If we're replacing every occurrence of the word `master`, so be it - but that's for another PR, and a naming discussion. Please remove it from this PR, and I'll approve.

File: optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java [hunk 5]
```diff
@@ -51,7 +51,7 @@
     }
 
     /**
-     * The source variables (masters) that trigger a change to this shadow variable (slave).
+     * The source variables (leaders) that trigger a change to this shadow variable (follower).
```
Comment (triceo) [inline optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java:54]: Please do not make changes to terminology without discussing with the developer team. 
Please remove this change from this PR, and let's dicsuss properly how to change this wording.

File: optaplanner-docs/src/main/asciidoc/ShadowVariable/ShadowVariable-chapter.adoc [hunk 6]
```diff
@@ -44,7 +44,7 @@ So if A references B, then B references A.
 image::ShadowVariable/bidirectionalVariable.png[align="center"]
 
 For a non-chained planning variable, the bi-directional relationship must be a many to one relationship.
-To map a bi-directional relationship between two planning variables, annotate the master side (which is the genuine side) as a normal planning variable:
+To map a bi-directional relationship between two planning variables, annotate the source side (which is the genuine side) as a normal planning variable:
```
Comment (triceo) [inline optaplanner-docs/src/main/asciidoc/ShadowVariable/ShadowVariable-chapter.adoc:47]: Dtto.

File: optaplanner-benchmark/src/main/resources/org/optaplanner/benchmark/impl/report/twitterbootstrap/js/jquery.js (hunk 3 above)
Comment (triceo) [reply optaplanner-benchmark/src/main/resources/org/optaplanner/benchmark/impl/report/twitterbootstrap/js/jquery.js:1264]: Good point, @rsynek. This is not even our source code, therefore subsequent updates would bring the word back in.

File: optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java (hunk 5 above)
Comment (rsynek) [reply optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java:54]: I checked this is the terminology on the `main` branch, assuming it has been properly reviewed and accepted there.

File: optaplanner-docs/src/main/asciidoc/ShadowVariable/ShadowVariable-chapter.adoc (hunk 6 above)
Comment (rsynek) [reply optaplanner-docs/src/main/asciidoc/ShadowVariable/ShadowVariable-chapter.adoc:47]: The same here - this is the wording we already have on the `main` branch.

**Comment by mbiarnes** [comment]
@triceo @rsynek 
https://github.com/kiegroup/optaplanner/pull/1466/files#diff-84bbe04464370ba19cff6d3a8d040f7f57b3f219b1cc92686486706a7d466de5R54 was copied from main branch

**Review by triceo — APPROVED** [review]

**Review by rsynek — APPROVED** [review]

File: optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java (hunk 5 above)
Comment (mareknovotny) [reply optaplanner-core/src/main/java/org/optaplanner/core/api/domain/variable/CustomShadowVariable.java:54]: so this change to keep is ok?

**Review by mareknovotny — COMMENTED** [review]
i am ok with all changes, but there were some request to not change other words than branch names and there is still that change kept so asking for reconfirmation

**Comment by triceo** [comment]
Yes, this is fine.

**Pull request merged**